packages/ozone/src/mod-service/index.ts

```diff
@@ -162,6 +168,25 @@ export class ModerationService {
       })
     }
 
+    // If multiple labels are passed, then only retrieve events where all those labels exist
```

Comment:
The description on this param says "only events where one of these labels were added are returned".  I could imagine either behavior working fine for us, but may need to tweak either the implementation or description so they're consistent.

Reply:
good catch, the mods told me that majority of cases, only 1 label is applied so I thought when multiple labels are passed here, we would be specifically looking for events where all labels exist. the implementation followed that so I needed to adjust the comment here.